.github/workflows/test.yml

```diff
@@ -127,7 +127,6 @@ jobs:
       - name: filter out DONTCOVER and e2e
         # yamllint disable
         run: |
-          (find . -type f -name '*.go' | xargs grep -l 'DONTCOVER') > exclude_list.txt
```

Comment:
This is removing filtering files that have the 'DONTCOVER' word which we don't have in the repo for now which makes the command fail. We can add this later if needed.